mutual_debt/main.py

```diff
@@ -0,0 +1,51 @@
+"""Mutual Debt Simplification
+
+Usage:
+  simplify-debts <data_file>
+  simplify-debts (-h | --help)
+
+Options:
+  -h --help     Show this screen.
+"""
+import json
+import sys
+
+from mutual_debt.mutual_debt_simplification import debt_list_to_graph, \
+    simplify_debt_graph, draw_graph
+
+
+def print_error(*args, sep=' ', end='\n'):
+    """ Prints values to the stderr stream """
+    print("ERROR:", *args, sep, end, file=sys.stderr)
+
+
+def main():
+
+    if len(sys.argv) != 2:
+        print(__doc__)
+        sys.exit(1)
+
+    if sys.argv[1] == '-h' or sys.argv[1] == '--help':
+        print(__doc__)
+        return
+
+    data_file = sys.argv[1]
```

Comment:
I think a default value would be nice so that you don't have to keep typing the file name if you use the default filename.

Reply:
Ok, then what should be the default name?
What do you think about *debts.json*?